packages/ui/src/components/va-slider/VaSlider.vue

```diff
@@ -515,9 +515,10 @@ export default defineComponent({
       }
     }
 
+    const pinPositionStep = computed(() => ((props.step) * 100) / (props.max - props.min))
```

Comment:
```suggestion
    const pinPositionStep = computed(() => (props.step * 100) / (props.max - props.min))
```

Reply:
IMHO, the percent formula (from school) looks like `part / whole * 100`, so my suggestion here ✌:

```suggestion
    const pinPositionStep = computed(() => (props.step / (props.max - props.min) * 100) 
```